accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/validators/ExternalGroupsInIsMemberOfValidator.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * (C) Copyright 2024 Netcentric AG.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package biz.netcentric.cq.tools.actool.validators;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import biz.netcentric.cq.tools.actool.configmodel.AcConfiguration;
+import biz.netcentric.cq.tools.actool.configmodel.AuthorizableConfigBean;
+import biz.netcentric.cq.tools.actool.configmodel.AuthorizablesConfig;
+import biz.netcentric.cq.tools.actool.configmodel.GlobalConfiguration;
+import biz.netcentric.cq.tools.actool.history.impl.PersistableInstallationLogger;
+import biz.netcentric.cq.tools.actool.validators.exceptions.InvalidExternalGroupUsageValidationException;
+
+@Component(service = ExternalGroupsInIsMemberOfValidator.class)
+public class ExternalGroupsInIsMemberOfValidator {
+    private static final Logger LOG = LoggerFactory.getLogger(ExternalGroupsInIsMemberOfValidator.class);
+
+    public void validateIsMemberOfConfig(AcConfiguration acConfiguration, PersistableInstallationLogger installLog,
+            GlobalConfiguration globalConfiguration) throws InvalidExternalGroupUsageValidationException {
+        List<String> externalGroupsValidationResults = checkIsMemberOfConfigsOfAllAuthorizables(acConfiguration);
+        if (!externalGroupsValidationResults.isEmpty()) {
+
+            externalGroupsValidationResults.stream().forEach(m -> installLog.addWarning(LOG, m));
+
+            String validationMsg = "Found " + externalGroupsValidationResults.size() + " group(s) that use external groups in isMemberOf. ";
```

Comment:
This may be user's as well! Again I would emit it the other way around:
`Found x authorizables as member of external group y`. Also the message should include which external group that is and which authorizables are members.

Reply:
line 38 adds each individual occurrence to the execution log. 

The recommendation is to use `isMemberOf` (also at this stage member values that could be moved to the other side of the relationship have been moved via https://github.com/Netcentric/accesscontroltool/blob/b60fd036ae71cee44cfb83916a9e388d18e6cccb/accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/YamlConfigurationMerger.java#L212

Reply:
> This may be user's as well! 

fixed via https://github.com/Netcentric/accesscontroltool/pull/735/commits/77fed31932f214418029d95975e45b38b0be1138